### ISSUE TITLE
Cannot add member in 3.1.0

### ISSUE THREAD

**Issue by w0k3** [m00re/taiga-docker#1]
Hi,

I'm not really sure if it is a configuration, image or platform problem, but I am unable to add members to projects in taiga-docker 3.1.0-ldap, in 3.0.0 it works just fine - in the same environment. In 3.1.0-ldap (using default docker-compose  file) the screen is blank with just "New Member" text in the center. Browser console throws following error:

```
Error: [$injector:unpr] Unknown provider: tProvider <- t <- tgSuggestAddMembersDirective
http://errors.angularjs.org/1.5.5/$injector/unpr?p0=tProvider%20%3C-%20t%20%3C-%20tgSuggestAddMembersDirective
t/<@http://localhost/v-1489162899787/js/libs.js:9:31649
nt/x.$injector<@http://localhost/v-1489162899787/js/libs.js:10:20469
r@http://localhost/v-1489162899787/js/libs.js:10:19393
nt/$<@http://localhost/v-1489162899787/js/libs.js:10:20552
r@http://localhost/v-1489162899787/js/libs.js:10:19393
i@http://localhost/v-1489162899787/js/libs.js:10:19693
a@http://localhost/v-1489162899787/js/libs.js:10:19895
T/</<@http://localhost/v-1489162899787/js/libs.js:10:26269
r@http://localhost/v-1489162899787/js/libs.js:10:66
T/<@http://localhost/v-1489162899787/js/libs.js:10:26238
a@http://localhost/v-1489162899787/js/libs.js:10:19999
a/<@http://localhost/v-1489162899787/js/libs.js:10:18151
a@http://localhost/v-1489162899787/js/libs.js:10:19999
nt/$<@http://localhost/v-1489162899787/js/libs.js:10:20572
r@http://localhost/v-1489162899787/js/libs.js:10:19393
Q@http://localhost/v-1489162899787/js/libs.js:11:4930
U@http://localhost/v-1489162899787/js/libs.js:10:30124
q@http://localhost/v-1489162899787/js/libs.js:10:29477
q@http://localhost/v-1489162899787/js/libs.js:10:29649
ne/<@http://localhost/v-1489162899787/js/libs.js:11:6349
a@http://localhost/v-1489162899787/js/libs.js:12:3271
s/<@http://localhost/v-1489162899787/js/libs.js:12:3443
xn/this.$get</p.prototype.$eval@http://localhost/v-1489162899787/js/libs.js:12:10956
xn/this.$get</p.prototype.$digest@http://localhost/v-1489162899787/js/libs.js:12:9460
xn/this.$get</p.prototype.$apply@http://localhost/v-1489162899787/js/libs.js:12:11241
ua[t]</<.compile/</<@http://localhost/v-1489162899787/js/libs.js:14:15138
oe.event.dispatch@http://localhost/v-1489162899787/js/libs.js:5:1949
oe.event.add/g.handle@http://localhost/v-1489162899787/js/libs.js:5:59
```

Every request returned 200, the last one (GET /api/v1/users/5/contacts?exclude_project=1) returned empty array, if that might cause the issue. I should be still able to invite users using e-mail address, right?

**Comment by m00re** [comment]
Hi. I can't say what the source of this issue is. I personally use Taiga to manage my own personal development projects. I don't have multiple users and have never tried the "invite user by email" feature. If you tell me how exactly to reproduce your issue, I will spent some time over the weekend to investigate it. In the meantime you can check the issue of the Taiga project (https://github.com/taigaio/taiga-front) to see whether someone else has a similar issue with the 3.1.0 release.

**Comment by w0k3** [comment]
Thank you for reply. The "New member" form should be accessible through any project's Admin page -> Members (/project/<projectname>/admin/memberships) -> green "+ NEW MEMBER" button in top left corner.

I've tried to search the issue; but I didn't find too much info, so I thought the issue is specific to taiga-docker. Someone claimed "downgrade to Ubuntu 16.04" fixed the issue (sounds unlikely): https://www.reddit.com/r/homelab/comments/5yyjlo/taiga_cant_add_member/

Today I've found this thread indicating that the code needs an update: https://groups.google.com/forum/#!topic/taigaio/X1NCdDNNBVU Am I able to pull a newer release in this container to give it a try?

**Comment by huksley** [comment]
Hello!

I am experiencing the same problem.

1. Docker compose install by your script (inside debian jessie)
2. Create project by admin (private)
3. Go to project admin panel
4. Members
5. Add new member
6. White screen
![image](https://cloud.githubusercontent.com/assets/1594349/26726528/146b312c-47ac-11e7-82d0-d90d8b16034e.png)

**Comment by m00re** [comment]
Hi. I just investigated the issue. I can reproduce this behaviour, and it seems as if the 3.1.0 release of Taiga has a bug here. I compared the implementation of the Angular directive **tgSuggestAddMembersDirective** in Taiga release 3.1.0 with the latest one in the master branch. As you can see [here](https://github.com/taigaio/taiga-front/commits/master/app/modules/invite-members/suggest-add-members/suggest-add-members.directive.coffee), the bug was fixed after the 3.1.0 release.

I will not release a new version of this Docker image unless Taiga.io has published a new release of the software. But you can try to fix the issue yourself by changing the Dockerfile to use the latest **stable** branch of Taiga instead of the latest release version (see https://github.com/taigaio/taiga-front-dist).

**Comment by m00re** [comment]
I will close this issue, as the issue belongs to the latest Taiga software release, and not to the Docker recipe provided here.
